tests/ui/boxed_local.rs

```diff
@@ -202,3 +203,8 @@ mod issue4804 {
         fn foo(x: Box<u32>) {}
     }
 }
+
+#[expect(clippy::boxed_local)]
```

Comment:
Shouldn't this be on the argument?

Reply:
I wasn't sure, sometimes I forget that attributes also work on arguments ^^. I've updated the test.